src/auth/auth-get-domain.ts

```diff
@@ -0,0 +1,33 @@
+const thirdPartyDomainSTSUrlMappings: { [domain: string]: string; } = {
+  'bbk12.com': 'https://sts-sso.bbk12.com'
+};
+const defaultSTSUrl = 'https://s21aidntoken00blkbapp01.nxt.blackbaud.com';
+
+export class BBAuthGetDomain {
+
+  private static cacheDictionary: any = {};
+
+  public static isRegisteredDomain(domain: string = window.location.hostname): boolean {
+    if (domain in this.cacheDictionary) {
+      return this.cacheDictionary[domain].isRegistered;
+    }
+    let isRegistered = false;
+    Object.keys(thirdPartyDomainSTSUrlMappings).forEach((d) => {
+      let domainEndingCompare = d;
+      if (!domainEndingCompare.startsWith('.')) {
```

Comment:
Eliminate the check for the leading `.` and always append it when doing the comparison below.